host/3.0/buster/amd64/python/python36/python36-slim.Dockerfile

```diff
@@ -77,7 +91,17 @@ RUN apt-get update && \
     #  binutils
     apt-get install -y binutils && \
     #  OpenMP dependencies
-    apt-get install -y libgomp1
+    apt-get install -y libgomp1 && \
+    # mysql dependencies
+    apt-get install -y default-libmysqlclient-dev
```

Comment:
What does this do?

Reply:
this is mysql dependencies and it is already in dev.